src/main/kotlin/io/snyk/plugin/snykcode/core/SnykCodeIgnoreInfoHolder.kt

```diff
@@ -34,14 +34,22 @@
         val prjBasePath = project.basePath
         val gitignore = File("$prjBasePath/.gitignore")
         val dcignore = File("$prjBasePath/.dcignore")
-        if (!gitignore.exists() && dcignore.createNewFile()) {
-            val fullDcIgnoreText = this.javaClass.classLoader.getResource("full.dcignore")
-                ?.readText()
-                ?: throw RuntimeException("full.dcignore can not be found in plugin's resources")
-            dcignore.writeText(fullDcIgnoreText)
-            SnykBalloonNotificationHelper.showInfo(
-                "We added generic .dcignore file to upload only project's source code.", project
-            )
+        if (!gitignore.exists()) {
+            val genericDcIgnoreFileCreated = try {
```

Comment:
Should we just remove this and do it on first scan only? I don't see a down-side to having it only when scanning...

Reply:
We can... just a bit more code/flags to recognize first scan of project....
Also having it for any Code scan helps to reduce load to our backend even in case user delete (accidently) .dcignore between scans